### ISSUE TITLE
Tooltips should be persistent.

### ISSUE THREAD

**Issue by bmeersma** [OfficeDev/office-ui-fabric-react#11721]
Delay property for hoverable content (the tooltip) may encourage developers to create an experience that is not consistent with the "presistent" aspect of WCAG 1.4.13.

https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html

**Comment by xugao** [comment]
@bmeersma - to clarify, do you mean that user can pass a really large `closeDelay` prop value which makes content remain visible after dismiss? If so , can you suggest what you have in mind on how to fix this issue? I don't think we can remove this feature, but i think we can set a max limit on its value.

**Comment by bmeersma** [comment]
Sorry, I miss interpreted what the delay value controls. Closing the issue.
